src/OpenSpaceToolkit/Physics/Coordinate/Spherical/LLA.cpp

```diff
@@ -99,6 +101,13 @@ Length LLA::getAltitude() const
     return altitude_;
 }
 
+Length LLA::calculateDistanceTo(
+    const LLA& aLLA, const Length& anEllipsoidEquatorialRadius, const Real& anEllipsoidFlattening
```

Comment:
Could we make clang format put each arg on its own line (especially when adding a trailing `,` after the last one, like black does?

Reply:
Unfortunately this is not possible AFAIK: https://stackoverflow.com/questions/50660455/clang-format-always-break-all-parameters-one-per-line

:( Hopefully they add this capability in the future.

Reply:
The trailing comma works for when using bracket initialization:
```
return {
        Derived::Undefined(),
        Length::Undefined(),
        Real::Undefined(),
        Real::Undefined(),
        Real::Undefined(),
    };
```
But not for function arguments.

Reply:
Ok!